Front end tools

tomiir opened the pull request:
## Summary
Modified index.html and titles.scss to center text on title image.

## Screenshot
![title comparison](https://user-images.githubusercontent.com/12738697/43852113-d249dd1e-9b12-11e8-9cde-b0aab7e35b1a.png)

## Trello Card
https://trello.com/c/DMRDozUb

Commit:
Progress on Title component. Still trying to center text vertically

Commit:
Uploading updated index and scss files

Commit:
Update HTML and CSS on Titles

HarryPerez commented on index.html:
```diff
@@ -9,6 +9,8 @@
 </head>
 <body>
-  <h1 class="title">Hello World</h1>
+  <h1 class="title">
```
You will have a lot of titles in you app. I would write here something like this `main-title` or `tech-title`.

HarryPerez replied:
Also i would put this title inside a container.
For example:
<img width="453" alt="captura de pantalla 2018-08-08 a la s 15 16 03" src="https://user-images.githubusercontent.com/36163179/43856123-05204d1c-9b1e-11e8-9798-2c630ee46337.png">


The `body-container` should have styles like this:
```
.body-container {
  margin: 0 auto;
  max-width: 1280px;
  padding: 100px 40px;
  width: 100%;
}
```
* 1280px is a default screen width that we are used to use
* padding is used in order to separate the screen from the content

HarryPerez commented on scss/application.scss:
```diff
@@ -1,2 +1,3 @@
 @import 'equalizer';
 @import 'styles';
+@import "titles";
```
Use simple quotes `@import 'titles';` here

HarryPerez commented on scss/styles.scss:
```diff
@@ -1,3 +1,12 @@
-.title {
-  color: red;
-}
+$blue: #00ADEE;
```
You should have another files separating vars:
* `sizes.scss`
```
$nav-height: 70px;
$nav-logo-height: 30px;
$title-decoration-height: 90px;
$title-decoration-width: 70px;
$tech-logo-size: 100px;
$text-big: 24px;
$text-small: 12px;
$text-medium: 16px;
```

* `colors.scss`
```
$blue: #00ADEE;
$green: #BED23C;
$grey: #C4C4C4;
```

`Take this as a good practice`

HarryPerez commented on scss/titles.scss:
```diff
@@ -0,0 +1,20 @@
+$title-size: 30px;
+$title-font: "Roboto", sans-serif;
+.title {
+  color: black;
```
this is not needed

HarryPerez commented on scss/titles.scss:
```diff
@@ -0,0 +1,20 @@
+  position: relative;
+}
+
+.title::before {
```
You should put `::before` inside `.title {` definition
```
.title {
...etc
&::before {
   ... etc
  }
}
```

HarryPerez replied:
in `before`
try something like this:
```
 &::before {
    background: url('./assets/triangle.svg') no-repeat;
    content: '';
    width: xxx
    height: xxx
    position: absolute;
    top: -18px;
    left: -28px;
    z-index: -1;
  }
```

tomiir replied:
Is is OK to use position:absolute? How does that work when what we are positioning is part of a component? Wouldn't it stay at a fixed position, independent from where the component is located?

Commit:
Fixed issues  marked on PR

tomiir commented:
## Summary
Added container to Title Component.
Added corresponding padding to match reference image.
Changed class name to be more specific.
Included ::before inside title class definition.
Divided styles scss into colors and sizes.

## Screenshot

## Trello Card 
https://trello.com/c/DMRDozUb

HarryPerez commented on index.html:
```diff
@@ -9,6 +9,18 @@
+    <h1 class="tech-title">Front-End Tools</h1>
+  </div>
+
+  <div class = "tech">
```
this is not inside the scope of this card

HarryPerez commented on scss/application.scss:
```diff
@@ -1,2 +1,5 @@
+@import 'colors';
+@import 'sizes';
+@import 'titles';
+@import 'tech';
```
this is not inside the scope of this card

HarryPerez commented on scss/application.scss:
```diff
@@ -1,2 +1,5 @@
 @import 'equalizer';
-@import 'styles';
+@import 'colors';
```
You can write `@import 'https://fonts.google.com/specimen/Roboto?selection.family=Roboto'; Here to import this font

HarryPerez commented on scss/styles.scss:
```diff
@@ -1,3 +0,0 @@
-.title {
```
You can write here 
```
* {
    font-family: 'Roboto', sans-serif;
}
``` 
In order to use it in the whole app

HarryPerez commented on scss/titles.scss:
```diff
@@ -0,0 +1,27 @@
+
+
+.body-container {
+margin: 0 auto;
```
add tabulation here. Remember that we use identation. And the identation is one tab (Two empty spaces)
```
.body-container {
  margin: 0 auto;
  max-width: 1280px;
  padding: 100px 40px;
  width: 100%;
}
```

Commit:
Fixed issues in PR

Commit:
Fix2

Commit:
Deleted tech file and fixed index

HarryPerez commented on scss/sizes.scss:
```diff
@@ -0,0 +1,10 @@
+$text-small: 12px;
+$text-medium: 16px;
+$title-size: 30px;
+$title-font: "Roboto", sans-serif;
```
this is not needed, because you are using 
```
* {
    font-family: 'Roboto', sans-serif;
}
```

In another file

HarryPerez commented on scss/application.scss:
```diff
@@ -1,2 +1,6 @@
+@import 'colors';
+@import 'sizes';
+@import 'titles';
+@import 'https://fonts.google.com/specimen/Roboto?selection.family=Roboto';
```
You have to import this font before using it. So you have to import it in a higher level.
I would put it after `@import 'equalizer';`
```
@import 'equalizer';
@import 'https://fonts.google.com/specimen/Roboto?selection.family=Roboto';
```

HarryPerez commented on .gitignore:
```diff
@@ -0,0 +1,4 @@
+#Basura de scss
+application.css
+application.css.map
+.scss-cache
```
you have to write `.sass-cache` not `.scss-cache`

HarryPerez replied:
check this!

HarryPerez commented on scss/titles.scss:
```diff
@@ -0,0 +1,27 @@
+.tech-title {
+  color: black;
```
Use colors defined in the `colors.scss` file

Commit:
PR Fixes

Commit:
fixed gitignore

Commit:
Removed cache files

HarryPerez commented:
Great Work!!!!!!
